grpc/proto/th2_grpc_read_db/read_db.proto

```diff
@@ -34,10 +34,12 @@ message QueryRequest {
   AssociatedMessageType associated_message_type = 4;
   repeated QueryId before_query_ids = 5;
   repeated QueryId after_query_ids = 6;
+//  maybe add parent event id parameter
```

Comment:
If you want. I don't have any objects. But this parameter should be optional and we should attach the event to the component parent event ID if it is not set